python/lsst/meas/extensions/photometryKron/kronLib.i

```diff
@@ -37,15 +37,39 @@ Interface to Kron magnitudes
 #include "lsst/afw/detection/Footprint.h"
 #include "lsst/afw/image/Utils.h"
 #include "lsst/afw/geom/Point.h"
-#include "lsst/meas/algorithms.h"
-#include "lsst/meas/extensions/photometryKron.h"
+#include "lsst/afw/math.h"
+#include "lsst/afw/image.h"
+#include "lsst/afw/detection.h"
+#include "lsst/afw/table.h"
+#include "lsst/meas/base.h"
+#define PY_ARRAY_UNIQUE_SYMBOL LSST_MEAS_BASE_NUMPY_ARRAY_API
```

Comment:
Yes, these need to be unique across all extension models.  @jdswinbank's suggestion is correct.  (And by the way, to get my attention here on GitHub, use @TallJimbo, but I was already watching this PR anyway).